### ISSUE TITLE
Running `nuxt build` when `@prisma/nuxt` is listed in the modules in `nuxt.config.ts` deletes other nuxt modules

### ISSUE THREAD

**Issue by damisparks** [prisma/nuxt-prisma#65]
The issue is still happening. It is related to https://github.com/prisma/nuxt-prisma/issues/22.
However, it stated that it was fixed on the PR https://github.com/prisma/nuxt-prisma/pull/62
The update is available in [v0.1.1](https://github.com/prisma/nuxt-prisma/releases/tag/v0.1.1) as stated here https://github.com/prisma/nuxt-prisma/pull/62#issuecomment-2503235451 

## Recommendation

Use [option B](https://www.prisma.io/docs/orm/more/help-and-troubleshooting/help-articles/prisma-nuxt-module#option-b-libprismats) as workaround as suggested in the closed issue https://github.com/prisma/nuxt-prisma/issues/22
Option B works for me. Hence, I got rid of the `@prisma/nuxt` module.

**Comment by ankur-arch** [comment]
Hey @damisparks 👋,

Thanks for reporting this issue! Could you try out the dev release @prisma/nuxt@0.0.0-experimental-692d1d6 and let us know if it resolves the problem for you?

Looking forward to your feedback!

**Comment by minhlucvan** [comment]
I am facing the same issue, tried `@prisma/nuxt@0.0.0-experimental-692d1d6` but no luck

**Comment by ankur-arch** [comment]
Thanks, @minhlucvan! 🙏 Could you let me know which package manager you're using?

Additionally, would it be possible for you to share a reproduction repo or example so I can test for a fix on my end?

**Comment by ankur-arch** [comment]
Hey folks 👋,

@minhlucvan, are you still encountering this issue with the latest version (`v0.2.0`)?

**Comment by damisparks** [comment]
@mhessdev how is this issue labelled as **not planned**?

**Comment by mhessdev** [comment]
Yup, just like @jharrell  said—we tried multiple times to follow up and get more information, but we couldn’t reproduce the issue, so we closed it. If we receive more details and a proper reproduction, I’d be more than happy to reopen this. @damisparks

**Comment by damisparks** [comment]
@mhessdev I was out on vacation. I will send a reproduction as soon as possible. 
However, it does not mean an issue should be closed as planned with a single comment.
I would suggest "**Closing issue due to lack of activity.**" as a closing comment.

**Comment by ankur-arch** [comment]
Hey @damisparks 👋 , let me know when you have a reproduction.
